### PR TITLE
Brig Area Update/Refactor

### PR DESCRIPTION
**Pull request by ItsVyzo**
# About the pull request

Updates Brig Areas to be more concise with new design

Also alters dropship hijack crash areas

# Explain why it's good for the game

People in X area of brig, hiding/dead/dying etc. are much easier to find. Also Mapper OCD

Dropship doors landing in space = bad

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl: LTNTS
add: brig areas: warden office, MP bunks, starboard hallway, interrogation room, evidence storage
spellcheck: changes brig surgery to brig medical
code: removed some brig areas from hijack, added other brig areas
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Review by Nanu308 — CHANGES_REQUESTED** [review]
New Areas but no placed APC's for those areas and rooms that were added?

**Comment by ItsVyzo** [comment]
> New Areas but no placed APC's for those areas and rooms that were added?

I need to stop forgetting things q=q

fixing

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: maps/map_files/USS_Almayer/USS_Almayer.dmm
Comment (Nanu308) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:1]: Needs the rewire semiotic added for the new APC's

**Review by Nanu308 — CHANGES_REQUESTED** [review]

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Nanu308 — APPROVED** [review]
lgtm

**Pull request merged**